components/metadata-service/cmd/metadataservice/metadataservice.go

```diff
@@ -198,11 +198,11 @@ func readMinioAccessConfiguration() (string, string, apperrors.AppError) {
 	secretAccessKey, foundSecret := os.LookupEnv(MinioSecretAccessKey)
 
 	if !foundId && !foundSecret {
-		return "", "", apperrors.Internal("%s and %s environment variables not set", MinioAccessKeyIdEnv, MinioSecretAccessKey)
+		return "", "", apperrors.Internal("%s and %s environment variables: could not find", MinioAccessKeyIdEnv, MinioSecretAccessKey)
```

Comment:
So I guess this error shows when two env vars are missing. Wouldn't it be better to say what are the missing vars? 

could not find -> not found

(I guess "MinioAccessKeyIdEnv, MinioSecretAccessKey" indicates the missing vars. but I don't know how it's shown in the error message)

Reply:
Missing vars are described with the consts. They are declared [here](https://github.com/kyma-project/kyma/blob/a995667644418138d1de9ff24e6603be819383ba/components/metadata-service/cmd/metadataservice/metadataservice.go#L34). I think they describe these env vars in a good way. They are just the names.